src/Generator/Changelog/Changeset.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace Mill\Generator\Changelog;
+
+use Mill\Generator\Traits\ChangelogTemplate;
+
+abstract class Changeset
+{
+    use ChangelogTemplate;
+
+    /**
+     * Get a changelog entry for a changeset that was added into, or removed from, the API.
+     *
+     * @param string $definition
```

Comment:
Might not be a problem when looking at the whole, but I needed to do some digging to find an example of a definition, It would be nice to have it a better explanation in these headers.